packages/react-instantsearch-hooks-server/src/__tests__/getServerState.test.tsx

```diff
@@ -421,7 +417,6 @@ describe('getServerState', () => {
               novalidate
         >
           <input class="ais-SearchBox-input"
-                 type="search"
```

Comment:
I removed this attribute because the attributes order differed in the React 17 and React 18 snapshots.